src/resources/js/rsvp.js

```diff
@@ -62,3 +62,10 @@ var tribe_tickets_rsvp = {
 		my.init();
 	} );
 })( jQuery, tribe_tickets_rsvp );
+
+jQuery(document).ready(function( $ ) {
+    $('.toggle').click(function() {
+        $(this).toggleClass( 'on' );
+        $(this).next('.attendee-meta-details').slideToggle();
+    });
+});
```

Comment:
A few things on this:

1. The logic in here should be relocated to `my.init()` rather than being a separate jQuery document.ready.
1. Perhaps also be a bit more specific with the selector.
1. The `next()` call places a pretty heavy reliance on HTML order - which is a bit more brittle than heading up to a parent and finding `.attendee-meta-details`

Maybe something like this?

```js
$( '.attendee-meta-row .toggle' ).on( 'click', function() {
  $( this )
    .toggleClass( 'on' )
    .siblings( '.attendee-meta-details' )
    .slideToggle();
} );
```